CONTRIBUTING.md

```diff
@@ -0,0 +1,176 @@
+# Contributing
+
+Contributing to this repository, we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within the project.
+
+## Issue tracker
+
+Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the [issue tracker](https://github.com/ansible-collections/community.mysql/issues).
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction initially and potentially save a lot of time and effort.
+Also somebody can already start discussing or working on implementing the same or similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be worth, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly
+
+We assume you use Linux as a work environment (you can use a virtual machine as well).
+
+1. Install ``docker``, launch it.
+
+2. Clone the [ansible-core](https://github.com/ansible/ansible) repository:
+```bash
+git clone https://github.com/ansible/ansible.git
+```
+3. Go to the cloned repository and prepare the environment:
+```bash
+cd ansible && source hacking/env-setup
+pip install -r requirements.txt
+cd ~
+```
+4. Create the following directories in your home directory:
+```bash
+mkdir -p ~/ansible_collections/community/mysql
+```
+
+5. Fork the ``community.mysql`` repository via the GitHub web interface.
+
+6. Clone the forked repository from your profile to the created path:
+```bash
+git clone https://github.com/YOURACC/community.mysql.git ~/ansible_collections/community/mysql
```

Comment:
Doesn't it make more sense to clone via ssh and not via https? Pushing things with https is pretty annoying :)

Reply:
Yeah, agreed that ssh is much better option generally.
Wouldn't a newbie have to generate SSH and do other necessary steps for that? It's not hard to set up but can be painful for complete newbies IMO. I think it could be made later, after a coder's first PR merged and they're happy:)
I'll add the option!

Reply:
done